packages/components/src/components/alertchip/alertchip.component.ts

```diff
@@ -0,0 +1,63 @@
+import { CSSResult, html } from 'lit';
+import { property } from 'lit/decorators.js';
+import styles from './alertchip.styles';
+import { Component } from '../../models';
+import { DEFAULTS } from './alertchip.constants';
+import { getAlertIcon } from './alertchip.utils';
+import type { VariantType } from './alertchip.types';
+import Buttonsimple from '../buttonsimple/buttonsimple.component';
+
+/**
+ * mdc-alertchip component is an interactive chip that consumers can use to represent an alert.
+ *
+ * - It supports a leading icon along with label.
+ * - It supports 5 variants of alerts - neutral, warning, error, success, and informational
+ *
+ * This component is built by extending Buttonsimple.
+ *
+ * @tagname mdc-alertchip
+ *
+ * @dependency mdc-icon
+ * @dependency mdc-text
+ */
+class Alertchip extends Buttonsimple {
+  /**
+   * The variant of the alertchip. It supports 5 variants
+   * - neutral
+   * - warning
+   * - error
+   * - success
+   * - informational
+   *
+   * @default neutral
+   */
+  @property({ type: String }) variant: VariantType = DEFAULTS.VARIANT;
+
+  /**
+   * The visible label text of the alertchip.
+   *
+   * We recommend limiting the <b>maximum length of the label text to 20 characters</b>,
+   * including empty spaces to split words.
+   */
+  @property({ type: String }) label = '';
```

Comment:
If you set the label as an empty string, then by default it will show as <mdc-alertchip label="" />, instead of if you set it to undefined, then it will show as <mdc-alertchip />, the label will be set only if it's defined by the consumer.
```suggestion
  @property({ type: String }) label?: string;
```

Reply:
Could you please clarify this?

Reply:
As the label is a mandatory field, we can't make it optional.